CountryApp/CountryApp/CountriesModel/CountriesModel.h

```diff
@@ -16,7 +16,7 @@ typedef void (^CountryManagerFailure)(NSString *failureReason, NSInteger statusC
 
 @interface CountriesModel : NSObject
 
-- (void)getCountriesForRegion:(NSString *) region success:(CountryManagerSuccess)success failure:(CountryManagerFailure)failure;
+- (void)getCountriesForRegion:(NSString *) region andSubRegion:(NSString *) subregion success:(CountryManagerSuccess)success failure:(CountryManagerFailure)failure;
```

Comment:
This is not good idea to make 2 different calls with 1 method

Reply:
It is only single call. In all cases 'region' and 'subregion' parameters is mandatory.